modules/log6x-code-samples.adoc

```diff
@@ -0,0 +1,119 @@
+// Module included in the following assemblies:
+//
+// observability/logging/log6x-clf.adoc
+
+
+:_mod-docs-content-type: REFERENCE
+[id="log6x-code-samples"]
+= Code Samples
+
+[source,yaml]
+----
+# Metadata Section: Defines the basic identification of the resource
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: logging  # Name of the ClusterLogForwarder resource
+  namespace: openshift-logging  # Namespace where the resource is located
+
+# Spec Section: Configures the audit policy with rules for logging
+rules:
+  # Rule Section: Specifies the actions that should be logged
+  - level: Metadata  # Log request metadata (including URI, verb, and user) for matched requests
+    verbs: ["create", "update", "patch", "delete"]  # Specify the HTTP verbs to audit
+    resources:
+      - group: ""  # Empty string refers to the core API group
+        resources: ["pods"]  # Apply the rule to pod resources
+    namespaces: ["default", "kube-system"]  # Limit the rule to specific namespaces
+    omitStages: ["RequestReceived"]  # Exclude specific audit stages from logging
+
+  # Another Rule Section: Logging all other requests at the 'None' level
+  - level: None  # Do not log requests that do not match any other rules
+
+# OmitStages Section: Specifies stages of request processing to be excluded from audit logs
+omitStages:
+  - "RequestReceived"  # Exclude the initial request receipt from audit logs
+----
+
+
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: logging
+  namespace: openshift-logging
+
+spec:
+  collector:
+    nodeSelector:
+      kubernetes.io/os: linux 
```

Comment:
In the CLF is not possible to define .spec.collector.* as it's part of the clusterLogging Kind, not from the `clusterLogforwarder`. Then, this example for the clusterLogForwarder is broken.

Also, it's missing defining the clusterLogging and the examples for it and setting limits as a good practice. This was requested in documentation bug - https://issues.redhat.com/browse/OBSDOCS-856. The reason is because of if not limits, it can bring down a node as using all the memory as visible in https://issues.redhat.com/browse/LOG-4745. But also, if it's added tolerations to the nodes, you also need to create a clusterLogging CR for being able to have running the collectors in all the nodes.

Also, in parallel, if https://issues.redhat.com/browse/LOG-4745 is implemented for Logging 6.0. We should indicate that if the user doesn't set limits, what they are these default limits

Reply:
Provide an example:
```
spec:
  collector:
    nodeSelector:
      kubernetes.io/os: linux
    resources:
      limits:
        memory: 2Gi
      requests:
        cpu: 1
        memory: 2Gi
    tolerations:
    - effect: NoSchedule
      key: node-role.kubernetes.io/master
      operator: Exists
    - effect: NoSchedule
      key: node.kubernetes.io/disk-pressure
      operator: Exists
```